bundles/paikkatietoikkuna/coordinatetransformation/resources/css/filesettings.css

```diff
@@ -24,17 +24,17 @@
     float: right;
     padding-left: 3em;
 }
-.file-select {
+.settingsSelect {
     float: left;
 }
-.file-select > select {
+.settingsSelect > select {
     float: right;
     width: 12em;
 }
-.cancel {
+.coordinate-form-cancel {
```

Comment:
I meant something like:

     .cancel -> .oskari-flyout.coordinatetransformation .cancel

and for all the styles in this file :)